### PR TITLE
Fix logger command

### PR DESCRIPTION
**Pull request by chloepugh**
The original command was missing quotation marks around a string

### CODE REVIEW

**Review by Th368MoJ — CHANGES_REQUESTED** [review]

File: source/documentation/appendix-docs/botor.md [hunk 1]
```diff
@@ -255,8 +255,7 @@ For further information consult the
 _Note:_ To turn off the debugging warnings found within the `botor` library,
 please use the following:
 ```r
```
[the remark this answers is not in the document]
Comment (chloepugh) [reply source/documentation/appendix-docs/botor.md:257]: It looks like all the code block in this file use ```r - should I remove for all of them?

File: source/documentation/data-docs/amazon-s3.md [hunk 2]
```diff
@@ -154,9 +154,20 @@ which use `renv` for managing environments. It requires the Python package
...
+renv::init()
+ 
+## or if you already have renv set, don't `init` the `renv` at all.
+
+## then, go ahead with the botor installation:
+renv::use_python() ## at the prompt, choose to use python3
 renv::install('reticulate')
+ 
+## Restart the session (Ctrl+Alt+F10 on a windows machine). And then:
+
```
Comment (Th368MoJ) [inline source/documentation/data-docs/amazon-s3.md:170]: Much clearer now

File: source/documentation/github.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ To create an SSH key in JupyterLab, follow the steps below:
 
 ### Add the SSH key to GitHub
 
-To add the SSH key to GitHub, you should follow the guidance [here](https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account).
+To add the SSH key to GitHub, you should follow the guidance [here](https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account). If you are only migrating your environment, you will only need to add your SSH key and set up the github user name and email, after which you should be able to use your old repositories. To try to clone them, follow [this guidance](https://user-guidance.services.alpha.mojanalytics.xyz/github.html#step-1-navigate-to-your-platform-r-studio-and-make-a-copy-of-the-github-project-in-your-r-studio) for R, and [this guidance](https://user-guidance.services.alpha.mojanalytics.xyz/github.html#command-line) for Python.
```
Comment (Th368MoJ) [inline source/documentation/github.md:66]: Can you turn this into paragraphs.

Comment (Th368MoJ) [reply source/documentation/github.md:66]: What are you referencing here with the `environment` bit?

**Comment by Th368MoJ** [comment]
@chloepugh your changes look good FYI. I'll merge everything when Amit's changes are approved

File: source/documentation/github.md (hunk 3 above)
Comment (DataStrategist) [reply source/documentation/github.md:66]: @Th368MoJ Sorry, this completely fell off my radar. The environment bit was attempting to clarify the steps required for the migration... but probably don't help?

**Pull request merged**